### PR TITLE
Descriptions updated for Fell Types collection

### PR DESCRIPTION
**Pull request by davelab6**
Thanks to @asantewa!

### CODE REVIEW

**Comment by davelab6** [comment]
@m4rc1e please take a look, I went over these line by line and they are golden but I may have missed something :)

**Comment by asantewa** [comment]
@googlebot I consent

**Comment by m4rc1e** [comment]
Thank you @asantewa for all the work you've done on these. They're a significant improvement!

I'll take a closer look next week.

File: ofl/imfelldwpicasc/DESCRIPTION.en_us.html [hunk 1]
```diff
@@ -1 +1,21 @@
...
\ No newline at end of file
+<p>
+IM Fell DW Pica SC (small caps) is one family within a revival project of all the Fell Types – a collection of types bequeathed by John Fell to the University of Oxford in 1686, who was Bishop of Oxford and Dean of Christ Church. 
+The fonts are all characterized by an unrefined look, where letterforms are intentionally uneven, which can be attributed to the strong fidelity of this particular revival process.
+</p>
+<p>
+All the typefaces are useful today for websites, books, essays, magazines and advertisements, and they come in different variations designed for different sizes of use. 
+IM Fell DW Pica SC is intended to be used at 12.5 points, suitable for body text.
```
Comment (m4rc1e) [inline ofl/imfelldwpicasc/DESCRIPTION.en_us.html]: Small cap fonts are not suitable for body text.

Comment (m4rc1e) [reply ofl/imfelldwpicasc/DESCRIPTION.en_us.html]: Perhaps you could say "IM Fell DW Pica SC can be used to improve the appearance of acronyms in body text."

File: ofl/imfellenglishsc/DESCRIPTION.en_us.html [hunk 2]
```diff
@@ -1 +1,21 @@
...
\ No newline at end of file
+<p>
+IM Fell English SC (small caps) is one family within a revival project of all the Fell Types – a collection of types bequeathed by John Fell to the University of Oxford in 1686, who was Bishop of Oxford and Dean of Christ Church. 
+The fonts are all characterized by an unrefined look, where letterforms are intentionally uneven, which can be attributed to the strong fidelity of this particular revival process.
+</p>
+<p>
+All the typefaces are useful today for websites, books, essays, magazines and advertisements, and they come in different variations designed for different sizes of use. 
+IM Fell English is intended to be used at 13.5 points, suitable for body text.
```
Comment (m4rc1e) [inline ofl/imfellenglishsc/DESCRIPTION.en_us.html]: Same issue as IM Fell DW Pica SC

**Pull request closed without merging**